FoxHound.Web/Controllers/CommentController.cs

```diff
@@ -0,0 +1,30 @@
+﻿using FoxHound.App.Blogs.CreateComment;
+using MediatR;
+using Microsoft.AspNetCore.Http;
+using Microsoft.AspNetCore.Mvc;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Threading.Tasks;
+
+namespace FoxHound.Web.Controllers
+{
+    [Route("api/[controller]")]
+    [ApiController]
```

Comment:
This shouldn't be needed as the BaseApiController contains this to keep consistency across all controllers.

Reply:
Removed.  I think I created the file as a controller which added those items in.